pkg-manager/core/test/hoistedNodeLinker/install.ts

```diff
@@ -39,6 +40,53 @@ test('installing with hoisted node-linker', async () => {
   expect(fs.realpathSync('node_modules/send')).toEqual(path.resolve('node_modules/send'))
 })
 
+test('installing with hoisted node-linker and list --json', async () => {
```

Comment:
don't put tests for `@pnpm/list` into core. If you need an e2e test, put it to the pnpm project.